### DISCUSSION TITLE
New Yarp and iCub releases available

### DISCUSSION THREAD

**Discussion by pattacini** [robotology/community#119] [Releases]
Dear @robotology/all-robotology-developers 

We have lately released new software binaries for **`yarp`** and **`icub-main`**. You'll find them available from the following resources:
- [**`Yarp 2.3.66`**](https://github.com/robotology/yarp/releases/tag/v2.3.66)
- [**`iCub 1.4.0`**](https://github.com/robotology/icub-main/releases/tag/v1.4.0)

Importantly, we have also introduced **new policies** for developing code in **`yarp`**, in particular regarding **branching**. Therefore, we warmly suggest you to look carefully at these [**contributing guidelines**](https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md).

Also, our intention is to reuse in the near future these new policies for the workflow of **`icub-main`**.

**Comment by pattacini** [comment]
cc @mbrunettini @drdanz

**Comment by traversaro** [comment]
Cool innovation of this release: Windows binaries for `icub-main` and its dependencies are available for 64-bit compilers!

**Comment by giodegas** [comment]
Just updated/re-built respective docker images on [dockerhub.com](http://dockerhub.com):
- [giodegas/yarp](http://hub.docker.com/r/giodegas/yarp)
- [giodegas/icub](http://hub.docker.com/r/giodegas/icub)

**Comment by pattacini** [comment]
Thanks @giodegas for maintaining [**Docker Hub**](https://hub.docker.com) repositories!

Just a comment on that: actually, the name of the robot is **iCub**, not **iCube** :wink:

**Comment by giodegas** [comment]
Thank you @pattacini and I apologize for the typo.
All fixed!

**Comment by pattacini** [comment]
Never mind! Thanks for the fix.
